### PR TITLE
Fixed some spelling and grammar (issue #29)

### PR DESCRIPTION
**Pull request by jerodev**
Could you please add the hacktoberfest label to this pull request?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,17 +27,17 @@
...
 
-* Clean the assets (build, images, css, js) in the project to maintain the directory organized
+* Clean the assets (build, images, css, js) in the project to keep the directory organized
 * Compress Images
 * Generate Sprites with .png
 * Generate Sprites with .svg and a .png to fallback
 * Analyze CSS with csslint
 * Prefix CSS with Autoprefixer
-* Compile Sass or Stylus (Catch the Stylus error and direct shows on the page, as in Sass)
+* Compile Sass or Stylus (Catch the Stylus and Sass errors and shows them directly on the page)
```
Comment (tiagoporto) [inline README.md]: The implementation its Just stylus, sass have this native.

File: README.md [hunk 2]
```diff
@@ -27,17 +27,17 @@
...
 * Concatenate And Minify Scripts
 * Analyze JavaScript with ESLint
-* Compile ES2015
-* Notify when tasks are complete
+* Compile to ES2015
```
Comment (tiagoporto) [inline README.md]: Compile es2015 to es5. I think compile es2015 is right.

**Review by tiagoporto — COMMENTED** [review]
Please, check the comments.

**Comment by jerodev** [comment]
Sorry for those, I have fixed these problems.

**Comment by curiouslychase** [comment]
I disagree with the changes in this PR.

> Notify when tasks are complete

was correct, every bullet point starts with a verb and changing "notify" to "notifications" changes this context from a feature.

**Missed:**

There is a mismatch in tense here:

> Monitors changes in the files and reload browser with BrowserSync

All of the other bullet points are the base form of the verbs you use, where Monitors implies a subject. 

I also recommend changing:

> Javascript tests with Jasmine and Karma

to "Test javascript with Jasmine & Karma"

File: README.md [hunk 3]
```diff
@@ -27,17 +27,17 @@
...
 * Generate Sprites with .svg and a .png to fallback
 * Analyze CSS with csslint
 * Prefix CSS with Autoprefixer
-* Compile Sass or Stylus (Catch the Stylus error and direct shows on the page, as in Sass)
+* Compile Sass or Stylus (Catch the Stylus errors and show them directly on the page, like with sass)
 * Concatenate And Minify Scripts
 * Analyze JavaScript with ESLint
 * Compile ES2015
-* Notify when tasks are complete
+* Notifications when tasks have completed
```
Comment (tiagoporto) [inline README.md]: I agree with @chaseadamsio

**Review by tiagoporto — COMMENTED** [review]
@jerodev there some conflicts. I cant't merge

**Pull request closed without merging**